docker/lr-winnowmap/Makefile

```diff
@@ -0,0 +1,14 @@
+VERSION = 2
+#TAG1 = quay.io/broad-long-read-pipelines/lr-canu:$(VERSION)
+#TAG2 = quay.io/broad-long-read-pipelines/lr-canu:latest
+TAG3 = us.gcr.io/broad-dsp-lrma/lr-winnowmap:$(VERSION)
```

Comment:
Another issue is if TAG1 and 2 are taken out, 3 and 4 should now be 1 and 2 accordingly.